### PR TITLE
feat: getting started guide tweaks

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Various tweaks that make the Getting Started guide much simpler. Includes several new scripts that make things easier without making them too abstract.

### CODE REVIEW

File: .envrc.example [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# NOTE: If you want to deploy contracts from scratch you MUST reload this
+#       variable to ensure the salt is regenerated and the contracts are
+#       deployed to new addresses (otherwise deployment will fail)
+export IMPL_SALT=$(openssl rand -hex 32)
+
+# Name for the deployed network
+export DEPLOYMENT_CONTEXT=getting-started
+
+# Optional Tenderly details for simulation link during deployment
+export TENDERLY_PROJECT=
```
Comment (tynes) [inline .envrc.example:47]: Pretty sure this is dead code now

File: .gitignore [hunk 2]
```diff
@@ -30,6 +30,7 @@ packages/contracts-bedrock/deployments/anvil
 .env
 .env*
 !.env.example
```
Comment (tynes) [inline .gitignore:32]: Can we delete `.env.example` from this now?

File: packages/contracts-bedrock/.envrc.example [hunk 3]
```diff
@@ -1,18 +0,0 @@
-# RPC for the network to deploy to
-export ETH_RPC_URL=
-
-# Sets the deployer's key to match the first default hardhat account
-export PRIVATE_KEY=ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
-
-# Name of the deployed network
-export DEPLOYMENT_CONTEXT=getting-started
```
Comment (tynes) [inline packages/contracts-bedrock/.envrc.example:8]: Are you sure it works without the deployment context set?

**Review by tynes — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (smartcontracts) [reply .gitignore:32]: I guess, seems like a different PR tho

File: .envrc.example (hunk 1 above)
Comment (smartcontracts) [reply .envrc.example:47]: It's still in `EnhancedScript`

**Pull request merged**